docs/content/docs/tutorials/rds-example.md

```diff
@@ -42,13 +42,13 @@ This guide assumes that you have:
 You can deploy the ACK service controller for Amazon RDS using the [rds-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart). You can download it to your workspace using the following command:
 
 ```bash
-helm pull oci://public.ecr.aws/aws-controllers-k8s/rds-chart --version=v0.0.15
+helm pull oci://public.ecr.aws/aws-controllers-k8s/rds-chart --version=v0.0.17
```

Comment:
This looks good but for future you can automate this part by fetching most recent tag from git repo and using that as version. (or using public-ecr describe-image-tags api.)